agents/ppo/ppo.py

```diff
@@ -0,0 +1,52 @@
+from agents.ppo.types import *
+from typing import List
+import torch
+
+
+class PPOAgent(TorchAgent, config_name='ppo'):
+    def __init__(self, actor: BaseActor, critic: BaseCritic, discount_factor: float, ratio_clip: float):
+        super().__init__()
+        self._actor = actor
+        self._critic = critic
+        self._discount_factor = discount_factor
+        self._ratio_clip = ratio_clip
+
+    @classmethod
+    def create_from_config(cls, config):
+        return cls(
+            actor=BaseActor.create_from_config(config['actor']),
+            critic=BaseCritic.create_from_config(config['critic']),
+            discount_factor=config['discount_factor'],
```

Comment:
Let's make params like `discount_factor` and `ratio_clip` parameters with default value and write here something like: `discount_factor=config.get('discount_factor', <default_value>),`